bftengine/src/preprocessor/messages/PreProcessReplyMsg.cpp

```diff
@@ -90,7 +90,8 @@ void PreProcessReplyMsg::setupMsgBody(const char* preProcessResultBuf,
     }
     {
       concord::diagnostics::TimeRecorder scoped_timer(*preProcessorHistograms_->signHash);
-      sigManager_->sign((char*)hash.data(), SHA3_256::SIZE_IN_BYTES, body() + headerSize, sigSize);
+      auto sigManager = SigManager::getInstance();
```

Comment:
Why take sigManager again?  It has been already taken at line 82. 
Or you can use it directly, like: sigSize = SigManager::getInstance()->getMySigLength();

Reply:
That's redundant. Thanks for seeing that.
Removed.